app/shared/components/Wallet/Status/Actions/Table/Row.js

```diff
@@ -0,0 +1,191 @@
+        const stakeToCpuDescription = (act.data.stake_cpu_quantity !== '0.0000 EOS') ? (
+          `${act.data.stake_cpu_quantity} ${t('actions_unstake_cpu_after_amount')}`
+        ) : '';
+
+        const stakeToNetDescription = (act.data.stake_net_quantity !== '0.0000 EOS') ? (
+          `${act.data.stake_net_quantity} ${t('actions_unstake_net_after_amount')}`
+        ) : '';
+
+        const stakeConnector =
+          (act.data.stake_cpu_quantity !== '0.0000 EOS' && act.data.stake_net_quantity !== '0.0000 EOS') ? (
+            ` ${t('actions_and_statement')} `
+          ) : '';
+
+        return (
+          <div>
+            <Icon
+              name="microchip"
+              size="large"
+              style={iconStyle}
+            />
+            <div>
+              { `${t('actions_unstaked_before_amount')} ${stakeToCpuDescription}${stakeConnector}${stakeToNetDescription}.` }
+            </div>
+          </div>
+        );
+      }
+      case 'undelegatebw': {
+        const unstakeToCpuDescription = (act.data.unstake_cpu_quantity !== '0.0000 EOS') ? (
+          `${act.data.unstake_cpu_quantity} ${t('actions_stake_cpu_after_amount')}`
+        ) : '';
+
+        const unstakeToNetDescription = (act.data.unstake_net_quantity !== '0.0000 EOS') ? (
+          `${act.data.unstake_net_quantity} ${t('actions_stake_net_after_amount')}`
+        ) : '';
+
+        const unstakeConnector =
+          (act.data.unstake_cpu_quantity !== '0.0000 EOS' && act.data.unstake_net_quantity !== '0.0000 EOS') ? (
+            ` ${t('actions_and_statement')} `
+          ) : '';
+
+        return (
+          <div>
+            <Icon
+              name="microchip"
+              size="large"
+              style={iconStyle}
+            />
+            <div>
+              { `${t('actions_staked_before_amount')} ${unstakeToCpuDescription} ${unstakeConnector} ${unstakeToNetDescription}.` }
+            </div>
+          </div>
+        );
+      }
+      case 'voteproducer': {
+        const {
+          data
+        } = act;
+
+        const {
+          producers
+        } = data;
+
+        const allButLastProducers = producers.slice(0, producers.length - 1).join(', ');
+
+        const andStatement = (producers.length > 1) ? (` ${t('actions_and_statement')}`) : '';
+
+        const lastProducer = producers.slice(-1);
+
+        return (
+          <div>
+            <Icon
+              name="check square"
+              size="large"
+              style={iconStyle}
+            />
+            <div>
+              { `${t('actions_voted')} ${allButLastProducers}${andStatement} ${lastProducer}.` }
+            </div>
+          </div>
+        );
+      }
+      default: {
+        return null;
+      }
+    }
+  }
+
+  render() {
+    const {
+      actionObject,
+      t
+    } = this.props;
+
+    const textStyle = {
+      fontFamily: '"Courier New", Courier, "Lucida Sans Typewriter", "Lucida Typewriter", monospace',
```

Comment:
Why are you setting a font here? I don't see a reason to have these strings be a formatted any differently than other text.

Reply:
I copied that over from the producers section 😄 I can remove it though